packages/fxa-auth-server/config/index.ts

```diff
@@ -854,7 +854,7 @@ const conf = convict({
     coupons: {
       enabled: {
         doc: 'feature flag for enabling coupons',
-        default: false,
+        default: true,
```

Comment:
We probably should have this default to false until we get the feature working in prod.